Library/Formula/signing-party.rb

```diff
@@ -201,3 +202,15 @@ def install
  # Organiser contact
  printf "%80s\n\n\n" "$org"
  # Title
+
+--- a/caff/caff	2016-02-01 21:59:58.000000000 +0100
++++ b/caff/caff	2016-02-01 22:00:42.000000000 +0100
```

Comment:
Wishlist Bug Report: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=813469
It would be nice if upstream could fix. But the problematic part is our perl version.